holidays/countries/argentina.py

```diff
@@ -309,15 +309,15 @@ def add_movable_holiday(dt: date, name: str) -> None:
         # Respect for Cultural Diversity Day or Columbus Day.
         # Status: In-Use.
         # First started in 1917 for Argentina.
-        # Have its name changed from 2010 onwards.
+        # Have its name changed as well as becoming movable from 2010 onwards.
```

Comment:
Consider rewording this:
```suggestion
        # In 2010 the holiday became movable and its name was changed.
```